src/Control/Carrier/Error/Either.hs

```diff
@@ -51,6 +51,9 @@ instance (Alternative m, Monad m) => Alternative (ErrorC e m) where
 -- | 'ErrorC' passes 'MonadPlus' operations along to the underlying monad @m@, rather than combining errors à la 'ExceptT'.
 instance (Alternative m, Monad m) => MonadPlus (ErrorC e m)
 
-instance (Algebra sig m, Effect sig) => Algebra (Error e :+: sig) (ErrorC e m) where
-  alg = ErrorC . alg . handleCoercible
+instance Algebra sig m => Algebra (Error e :+: sig) (ErrorC e m) where
+  -- NB: 'send' (& thus 'handleCoercible') can’t send sums, so we decompose the sum manually.
+  alg (L (L op)) = ErrorC (handleCoercible op)
+  alg (L (R op)) = ErrorC (handleCoercible op)
+  alg (R op)     = ErrorC (handleCoercible op)
```

Comment:
This is the one weird case: because `handleCoercible` uses `send`, and because `send` uses `Member`, and because `Member` can’t resolve sums of effects on the left, we have to decompose the sum here instead of doing just `alg = ErrorC . handleCoercible`.

I wouldn’t expect end users to have to do this very often, however, since it’s not very often you want to delegate _all_ of the effects you handle to something else (how I wish we could simply GND-derive the instance!), or that (as in this case), you’re handling multiple effects combined in a sum, let alone both.